models/migrations/migrations.go

```diff
@@ -413,6 +413,8 @@ var migrations = []Migration{
 	NewMigration("Add badges to users", createUserBadgesTable),
 	// v225 -> v226
 	NewMigration("Alter gpg_key/public_key content TEXT fields to MEDIUMTEXT", alterPublicGPGKeyContentFieldsToMediumText),
+	// v226 -> v227
+	NewMigration("Fix package semver field", fixPackageSemverField),
```

Comment:
I think a notice that this affects "conan" and "generic" packages specifically would be a good idea.
```suggestion
	NewMigration("Conan and generic packages do not need to be semantically versioned", fixPackageSemverField),
```